app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/controller/SearchController.java

```diff
@@ -112,6 +121,70 @@ public static SearchController getInstance() {
         return resultFuture;
     }
 
+    public CompletableFuture<ArrayList<OwnerListItem>> getAvailableOwners(String isbn) {
+        final CompletableFuture<ArrayList<OwnerListItem>> ownersFuture = new CompletableFuture<>();
+
+        index.searchAsync(new Query(isbn),
+                (content, error) -> {
```

Comment:
This looks like repeated code from searchBooks

Reply:
getBook(@NonNull String isbn)  in book controller will give you the book of the exact match isbn. This is fuzzy search so it might behave unexpected